eras/babbage/impl/src/Cardano/Ledger/Babbage/PParams.hs

```diff
@@ -350,32 +351,37 @@ babbagePParamsPairs ::
   PParamsHKD Identity era ->
   [a]
 babbagePParamsPairs pp =
-  uncurry (.=) <$> babbagePParamsHKDPairs (Proxy @Identity) pp
+  uncurry (.=)
+    <$> babbagePParamsHKDPairs (Proxy @Identity) pp
+      ++ [ "decentralization" .= Aeson.Null
+         , "extraPraosEntropy" .= Aeson.Null
+         , "minUTxOValue" .= Aeson.Null
```

Comment:
Why is this necessary? These have been deprecated and are no longer applicable in Babbage

Reply:
If we want to continue generating the same exact format as `cardano-cli` we would need to do this, because it currently uses the same data type for all eras. But maybe we don't have to as long as we can parse it, for example. @Jimbo4350, what do you think?